files/zh-cn/web/http/headers/set-cookie/index.md

```diff
@@ -22,178 +24,188 @@ slug: Web/HTTP/Headers/Set-Cookie
       <td>否</td>
     </tr>
     <tr>
-      <th scope="row">{{Glossary("Forbidden response header name", "禁止的响应标头")}}</th>
+      <th scope="row">{{Glossary("Forbidden response header name", "禁止修改的响应标头")}}</th>
       <td>是</td>
     </tr>
   </tbody>
 </table>
 
 ## 语法
 
-```
+```http
 Set-Cookie: <cookie-name>=<cookie-value>
+Set-Cookie: <cookie-name>=<cookie-value>; Domain=<domain-value>
 Set-Cookie: <cookie-name>=<cookie-value>; Expires=<date>
+Set-Cookie: <cookie-name>=<cookie-value>; HttpOnly
 Set-Cookie: <cookie-name>=<cookie-value>; Max-Age=<number>
-Set-Cookie: <cookie-name>=<cookie-value>; Domain=<domain-value>
+Set-Cookie: <cookie-name>=<cookie-value>; Partitioned
 Set-Cookie: <cookie-name>=<cookie-value>; Path=<path-value>
 Set-Cookie: <cookie-name>=<cookie-value>; Secure
-Set-Cookie: <cookie-name>=<cookie-value>; HttpOnly
 
 Set-Cookie: <cookie-name>=<cookie-value>; SameSite=Strict
 Set-Cookie: <cookie-name>=<cookie-value>; SameSite=Lax
 Set-Cookie: <cookie-name>=<cookie-value>; SameSite=None; Secure
 
-// Multiple attributes are also possible, for example:
+// 可以同时有多个属性，例如：
 Set-Cookie: <cookie-name>=<cookie-value>; Domain=<domain-value>; Secure; HttpOnly
 ```
 
 ## 指令
```

Comment:
Should we change original document to 'Directives' here?

Reply:
Actually, we could use "attributes", as this is used by the [spec](https://httpwg.org/specs/rfc6265.html#sane-set-cookie).

```suggestion
## 属性
```